backend/errors/domain.error.js

```diff
@@ -0,0 +1,13 @@
+class DomainError extends Error {
```

Comment:
Let's keep things simple and have only one level of error inheritance in our application for now.
Sorry, I didn't add some details about what part of the example from the article we need to use. 
Let's make this file `DatabaseError` and remove its inheritor.

Reply:
I removed DomainError from the chain and made this file DatabaseError. I kept the inheritance to the Base Error class though, as this contains the message field, which we need and is already implemented by Node.js.